.github/workflows/build-firmware.yaml

```diff
@@ -335,13 +339,6 @@ jobs:
       working-directory: ./firmware/
       run: ./gen_enum_to_string.sh
 
-      # depends on gen_live_documentation.sh for instance TS_TOTAL_OUTPUT_SIZE
-    - name: Generate Default config
-      if: ${{ env.skip != 'true' }}
```

Comment:
i have a feeling that this would break something major, let' start breaking the change apart

Reply:
there's always the scream test of "do it and see what screams"